CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+# This file helps Github doing automatic review requests for new PRs.
+# It should always list the active maintainers of certain add-ons.
+
+# As a fallback, if no specific maintainer is listed below, assign the PR to the repo maintainers team:
+*       @openhab/2-x-add-ons-maintainers
+
+# Add-on maintainers:
+/addons/ui/iconset/ @kaikreuzer
+/addons/ui/org.openhab.ui.basic/ @lolodomo @resetnow
+/addons/ui/org.openhab.ui.classic/ @lolodomo @resetnow
```

Comment:
@resetnow is imho not involved with the Classic UI, better put me there next to @lolodomo.